src/CMakeLists.txt

```diff
@@ -15,6 +15,19 @@ add_third_party(
   LIB libdouble-conversion.a
 )
 
+add_third_party(
```

Comment:
Json branch already has these changes. Why you just don't start from there

Reply:
I've started from the Json branch with your existing changes related to the JSON functionality, Despite that Github shows also your commits that are already part of  JSON branch.